### PR TITLE
Soko2 PR2 in Client 

### PR DESCRIPTION
**Pull request by solongoch**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -18,30 +20,34 @@ import Followers from './components/Layout/Followers';
...
+              <Route exact path='/signup' component={Signup} />
+              <Route exact path="/login" component={Login} />
+              <Route exact path="/home" component={Homepage} />
+              <Route exact path="/suggestion" component={Suggestion} />
+              <Route exact path="/edit" component={EditProfile} />
+              <Route exact path='/changepassword' component={ChangePassword} />
+              <Route exact path="/profile" component={Profile} />
+              <Route exact path="/single-post" component={SinglePost} />
+              <Route path='/create' component={CreatePost} />
+              <Route exact path='/followers' component={Followers} />
```
Comment (slpriya) [inline client/src/App.js]: For followers following and unfollow pls keep Master version

File: client/src/components/Layout/CreatePost.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import logoImage from "../../image/avatar.png";
 import '../css/createpost.css';
-import cloudniary from '../config/Key';
+import cloudinary from '../config/Keys';
```
Comment (slpriya) [inline client/src/components/Layout/CreatePost.js]: this line is removed. pls keep Master version

File: package.json [hunk 3]
```diff
@@ -25,7 +25,10 @@
     "mongoose": "^5.9.20",
     "passport": "^0.4.1",
     "passport-jwt": "^4.0.0",
+    "react-redux": "^7.2.0",
```
Comment (slpriya) [inline package.json]: I think you installed all redux related libraries in server-side. because I can see mongoose in this file.

File: server.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const {cloudinary} = require('./client/src/components/config/keys');
+const cloudinary = require('./client/src/components/config/keys');
```
Comment (slpriya) [inline server.js]: Keep Master version

**Review by slpriya — APPROVED** [review]
Check Comments

**Review by slpriya — APPROVED** [review]

**Review by Emily484 — COMMENTED** [review]
looks good

**Pull request merged**